src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,29 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and overriding `clone` method is not needed for entity classes. Hibernate handles entity duplication in the context of sessions, and manually cloning entities might lead to unexpected behavior and is not a common practice.